config/default/foia_wizard.settings.yml

```diff
@@ -99,52 +99,52 @@ messages:
   m38:
-    value: "<p>Select specific branch of the military to start a FOIA request</p>\r\n\r\n<ul>\r\n\t<li>DOD-Air ForceDOD FOIA.gov page</li>\r\n\t<li>DOD-Army FOIA.gov page</li>\r\n\t<li>DOD-Navy FOIA.gov page</li>\r\n\t<li>DOD-USMC FOIA.gov page</li>\r\n\t<li>DHS-USCG FOIA.gov page</li>\r\n\t<li>For National Guard records contact the Adjutant General of the state in which the member served: https://www.nationalguard.mil/Resources/State-Websites/</li>\r\n</ul>\r\n"
+    value: "<p>Select the specific branch of the military to start a FOIA request:</p>\r\n\r\n<ul>\r\n\t<li><a href=\"https://www.foia.gov/?id=22c43aec-3e80-4e9d-824e-bddc2a44cdf1&amp;type=component\">Air Force</a></li>\r\n\t<li><a href=\"https://www.foia.gov/?id=d579e5b4-a69a-4d5c-aa2c-467fe92c3021&amp;type=component\">Army</a></li>\r\n\t<li><a href=\"https://www.foia.gov/?id=f63858a3-e585-4807-b7ab-20c43db3fc3d&amp;type=component\">Navy</a></li>\r\n\t<li><a href=\"https://www.foia.gov/?id=59bd6238-3764-4106-97e9-a774a3ff1592&amp;type=component\">Marine Corp</a></li>\r\n\t<li><a href=\"https://www.foia.gov/?id=52ca77d2-005e-4231-9206-a2e7201a27e9&amp;type=component\">Coast Guard</a></li>\r\n\t<li><a href=\"https://www.nationalguard.mil/Resources/State-Websites/\">National Guard</a></li>\r\n</ul>\r\n"
     format: rich_text
   m39:
-    value: "<p>OMPFs can be requested using the eVetRecs online request form. Alternatively, you may submit Standard Form 180.</p>\r\n"
+    value: "<p>You can request Official Military Personnel Files using the <a href=\"https://www.archives.gov/veterans/military-service-records\">eVetRecs online request form</a>.</p>\r\n\r\n<p>You can also submit the <a href=\"https://www.archives.gov/veterans/military-service-records/standard-form-180.html \">Standard Form 180</a>.</p>\r\n"
     format: rich_text
   m40:
-    value: "<p>DD Form 214 Report of Separation can be requested using the eVetRecs online request form. Alternatively, you may submit Standard From 180.</p>\r\n"
+    value: "<p>You can request a DD Form 214 (Report of Separation) using the <a href=\"https://www.archives.gov/veterans/military-service-records\">eVetRecs online request form</a>.</p>\r\n\r\n<p>You can also submit the <a href=\"https://www.archives.gov/veterans/military-service-records/standard-form-180.html \">Standard Form 180</a>.</p>\r\n"
     format: rich_text
   m41:
-    value: "<ul>\r\n\t<li>MilConnect</li>\r\n\t<li>The following agencies may also have the records you seek:\r\n\t<ul>\r\n\t\t<li>VA/VBA FOIA.gov page</li>\r\n\t</ul>\r\n\t</li>\r\n</ul>\r\n"
+    value: "<p>You can access Veteran’s Benefits information through <a href=\"https://milconnect.dmdc.osd.mil/milconnect/ \">MilConnect</a>.</p>\r\n\r\n<p>The following agencies may also have the records you seek:</p>\r\n\r\n<ul>\r\n\t<li><a href=\"https://www.foia.gov/?id=7327fdbe-35ed-4381-bd2d-fc5b48ef7f40&amp;type=component \">Department of Veterans Affairs - Veterans Benefits Administration</a></li>\r\n</ul>\r\n"
```

Comment:
I suggest a file-wide find replace of `https://www.foia.gov/?id=` to `https://www.foia.gov/agency-search.html?id=` so these use their new URLs without a redirect.